pig/src/main/java/org/jboss/pnc/bacon/pig/impl/nvr/NvrListGenerator.java

```diff
@@ -54,10 +55,16 @@ public static boolean generateNvrList(Map<String, Collection<String>> checksums,
         }
 
         File srcFile = new File(outputDirectory, basename);
-        File destFile = new File(targetPath);
+        File destFile = targetPath.toFile();
 
+        if (destFile.isFile()) {
+            if (!destFile.delete()) {
+                throw new RuntimeException(
+                        "Failed to delete the previously created nvr list: " + destFile.getAbsolutePath());
+            }
+        }
         try {
-            FileUtils.copyFile(srcFile, destFile, true);
+            FileUtils.moveFile(srcFile, destFile);
```

Comment:
OK, this is why I changed it to `copyFile()` originally. I was having an issue with the `destFile` already existing, I think. As long as we're sure we actually want to move the file (rerunning should overwrite this file anyway)/

Reply:
With copying, the problem was that when the target directory was fixed, the nvr.txt was created there and polluted the release directory. 
Alternatively to moving we can create the file in some temp location and copy it but copying is a bit easier.

Reply:
There were two nvr txt files in the directory I think? I am sorry that the Build Finder API just doesn't allow you to change the output file name in the first place.

Reply:
Actually, you can call `report.setBaseFilename()` to set the name. It will automatically add the`.txt` extension though. Doesn't it make sense to write it directly? Then we can remove the call to `moveFile()` altogether.

Here's what it calls internally:

```
FileUtils.writeStringToFile(new File(this.outputDirectory, this.baseFilename + ".txt"), text, StandardCharsets.UTF_8, false);
```

Reply:
that simplifies things a lot!